memory-store.js

```diff
@@ -6,6 +6,21 @@ var compareTime = require('./compare-time')
  * It is good for tests, but not for server or client usage,
  * because it doesn’t save events to file or localStorage.
  *
+ * Think about this store as a basic store realization.
+ *
+ * Every Store class should provide three standard methods:
+ * add, get and remove.
+ *
+ * Every Store for client should provide two more methods:
+ * getSynced, setSynced.
+ *
+ * @param {object} [options] Store options.
+ * @param {number} [options.synced=0] Events with lower `added` time in current
```

Comment:
Yes, I think it can be useful (`setOptions`/`setProperties`/`setData`/`setExtra`) in future but as we don't have cases now – lets add this `*LatestSynced` methods and after we'll implement more database stores – we'll see what we could need and then change API. I think it will be clear before next version release and we'll change it to more general names later if we'll need it